app/konnect/dev-portal/customization/index.md

```diff
@@ -27,16 +27,24 @@ Here, you have the ability to modify the following:
 
 For details on the requirements for each customizable option, hover over the information (`i`) icon next to each item in the UI.
 
-
 ## Custom Developer Portal URL
 
 All Dev Portals have an auto-generated default Dev Portal URL. You can add a custom domain. When set up properly, users can access the Dev Portal from both the default URL and the custom URL. The {{site.konnect_short_name}} Kong Developer Portal generates an SSL certificate for your custom domain automatically.
 
+
 ### Prerequisites
 
 * A domain and access to configure domain's CNAME
 * Your organization's auto-generated default Dev Portal URL. For example, `https://kong121212.portal.konnect.konghq.com/`.
 
+### Supported top-level domains
```

Comment:
I don't think the table is rendering correctly here (the second column is missing). 

Also, I think this information would fit better as a list under the first bullet in the Prereq section? I'll suggest something so you can see what I mean.

Reply:
+1 on this as well. Either intro or prereq, but I think it would be more readable as a list.